pkg/publisher/util/utils.go

```diff
@@ -23,35 +24,49 @@ func NewPublisherProvider(
 	ctx context.Context,
 	storagePath string,
 	cm *system.CleanupManager,
-	ipfsConnect string,
+	cfg types2.PublishersConfig,
 	localConfig *types.LocalPublisherConfig,
 ) (publisher.PublisherProvider, error) {
-	noopPublisher := noop.NewNoopPublisher()
-	s3Publisher, err := configureS3Publisher(storagePath, cm)
-	if err != nil {
-		return nil, err
-	}
+	providers := make(map[string]publisher.Publisher)
 
-	localPublisher := local.NewLocalPublisher(ctx, localConfig.Directory, localConfig.Address, localConfig.Port)
+	// TODO(review): the NoopPublisher is a testing attribute, should it remain part of the default set of publishers?
+	// for now, lets keep everything the same for compatibility, else tests might fail.
+	providers[models.PublisherNoop] = noop.NewNoopPublisher()
```

Comment:
@wdbaruni should we continue to include the noop publisher as a default, or remove it and require tests to inject it?

Reply:
make sense if the change is trivial